api/v1alpha1/istio_structs.go

```diff
@@ -16,7 +16,7 @@ type Components struct {
 	// Pilot defines component configuration for Istiod
 	Pilot *IstioComponent `json:"pilot,omitempty"`
 	// IngressGateways defines component configurations for Istio Ingress Gateways
-	IngressGateways []*IstioComponent `json:"ingressGateways,omitempty"`
+	IngressGateway *IstioComponent `json:"ingressGateway,omitempty"`
```

Comment:
We need to update the documentation on this:
```
// IngressGateway defines component configurations for Istio Ingress Gateway
```

Reply:
thanks, fixed